### PR TITLE
Wip 345 add datasets metadata with file

### PR DESCRIPTION
**Pull request by LancelotMarti**

### CODE REVIEW

**Comment by PaulineMauryL** [comment]
## Coverage report


 

_This PR does not seem to contain any modification to coverable code._

File: server/requirements_streamlit.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 requests==2.32.0
-streamlit==1.37.0
```
Comment (PaulineMauryL) [inline server/requirements_streamlit.txt:2]: why not ? there is a security warning coming with 1.35.0

Comment (PaulineMauryL) [reply server/requirements_streamlit.txt:2]: https://github.com/dscc-admin-ch/lomas/security/dependabot/20

Comment (LancelotMarti) [reply server/requirements_streamlit.txt:2]: ah... but with 1.37 the streamlit app broke with the main page, [this one](https://discuss.streamlit.io/t/attributeerror-str-object-has-no-attribute-main-script-path/73585) so i revert back to 1.35.

I'll try with 1.39 that they released on the 01.10, maybe they fixed it

Comment (PaulineMauryL) [reply server/requirements_streamlit.txt:2]: oups, I didn't realize it has broken the page, it is better if it works indeed ^^

Comment (LancelotMarti) [reply server/requirements_streamlit.txt:2]: the issue is actually coming from the st-pages package, updating it to 0.5.0 fixes the problem :)

Comment (PaulineMauryL) [reply server/requirements_streamlit.txt:2]: might be good to add tests that the streamlit app has no bug, I'll have a look

Comment (PaulineMauryL) [reply server/requirements_streamlit.txt:2]: nice!

**Review by PaulineMauryL — APPROVED** [review]

File: server/requirements_streamlit.txt (hunk 1 above)
Comment (PaulineMauryL) [reply server/requirements_streamlit.txt:2]: (Seems possible https://docs.streamlit.io/develop/api-reference/app-testing i made a ticket)

Comment (LancelotMarti) [reply server/requirements_streamlit.txt:2]: good idea

**Pull request merged**